<div align="center"><span style="color:green"><h1> INTERNATIONAL STUDENT DATA </h1></span></div>
<br>
<div align="center"><span style="color:green"><h4> Rickardo Henry -- IS362 </h4></span></div>

We will begin by importing pandas and reading the .csv data. Reading the data is captured from [Github](https://github.com/).

In [1]:
import pandas as pd # Importing Module Called pandas for usage.

student_data = pd.read_csv('C:/Users/rickardo.henry/Desktop/IS362_Project2/Student_Data.csv')
#reading data from Github
student_data



Country             Region  Gender  Age-years        Handed  \
0            NaN                NaN     NaN        NaN           NaN   
1      Australia    New South Wales  Female       10.0  Right-handed   
2            NaN                NaN     NaN        NaN           NaN   
3      Australia           Victoria  Female       14.0  Ambidextrous   
4            NaN                NaN     NaN        NaN           NaN   
5    New Zealand           Auckland  Female       14.0   Left-handed   
6            NaN                NaN     NaN        NaN           NaN   
7             UK         South West  Female       15.0  Right-handed   
8            NaN                NaN     NaN        NaN           NaN   
9      Australia    New South Wales  Female       16.0  Right-handed   
10           NaN                NaN     NaN        NaN           NaN   
11     Australia  Western Australia    Male       11.0  Right-handed   
12           NaN                NaN     NaN        NaN           NaN   
13        Canada            Alberta  Female       12.0  Right-handed   
14           NaN                NaN     NaN        NaN           NaN   
15     Australia         Queensland    Male       15.0  Right-handed   
16           NaN                NaN     NaN        NaN           NaN   
17        Canada            Ontario  Female       14.0  Right-handed   
18           NaN                NaN     NaN        NaN           NaN   
19            UK      West Midlands    Male       13.0  Right-handed   
20           NaN                NaN     NaN        NaN           NaN   
21     Australia           Tasmania  Female       11.0  Right-handed   
22           NaN                NaN     NaN        NaN           NaN   
23   New Zealand           Auckland    Male       10.0  Right-handed   
24           NaN                NaN     NaN        NaN           NaN   
25   New Zealand           Auckland  Female       10.0  Right-handed   
26           NaN                NaN     NaN        NaN           NaN   
27        Canada            Ontario    Male       12.0  Right-handed   
28           NaN                NaN     NaN        NaN           NaN   
29        Canada            Ontario    Male       17.0  Right-handed   
..           ...                ...     ...        ...           ...   
970          NaN                NaN     NaN        NaN           NaN   
971          USA                 CA  Female       11.0  Right-handed   
972          NaN                NaN     NaN        NaN           NaN   
973          USA                 GA    Male       16.0  Right-handed   
974          NaN                NaN     NaN        NaN           NaN   
975          USA                 AZ  Female       16.0  Right-handed   
976          NaN                NaN     NaN        NaN           NaN   
977          USA                 SC    Male       15.0  Right-handed   
978          NaN                NaN     NaN        NaN           NaN   
979          USA                 NH    Male       17.0  Ambidextrous   
980          NaN                NaN     NaN        NaN           NaN   
981          USA                 CA  Female       17.0  Right-handed   
982          NaN                NaN     NaN        NaN           NaN   
983          USA                 CO  Female       15.0  Right-handed   
984          NaN                NaN     NaN        NaN           NaN   
985          USA                 CA  Female       17.0  Right-handed   
986          NaN                NaN     NaN        NaN           NaN   
987          USA                 GA    Male       17.0  Right-handed   
988          NaN                NaN     NaN        NaN           NaN   
989          USA                 PA    Male       16.0  Right-handed   
990          NaN                NaN     NaN        NaN           NaN   
991          USA                 IN    Male       18.0  Right-handed   
992          NaN                NaN     NaN        NaN           NaN   
993          USA                 NJ  Female      

<b><span style="color: #FF0000;"> Q1. Which country had the most international students ?</span></b>

In [7]:
student_data.groupby('Country').count()

Region  Gender  Age-years  Handed  Height_cm  Footlength_cm  \
Country                                                                    
Australia       170     170        170     170        170            170   
Canada          100     100        100     100        100            100   
New Zealand      74      74         74      74         74             74   
UK               56      56         56      56         56             56   
USA             100      99        100      99         95             89   

             Armspan_cm  Languages_spoken  Travel_to_School  \
Country                                                       
Australia           170               170               170   
Canada              100               100               100   
New Zealand          74                74                74   
UK                   56                56                56   
USA                  89                96                97   

             Travel_time_to_School  Reaction_time  Score_in_memory_game  \
Country                                                                   
Australia                      170            169                   167   
Canada                         100            100                   100   
New Zealand                     74             74                     0   
UK                              56             56                    56   
USA                             95             92                    89   

             Favourite_physical_activity  Importance_reducing_pollution  \
Country                                                                   
Australia                            170                            167   
Canada                               100                            100   
New Zealand                           74                             74   
UK                                    56                             56   
USA                                   94                             87   

             Importance_recycling_rubbish  Importance_conserving_water  \
Country                                                                  
Australia                             165                          166   
Canada                                100                          100   
New Zealand                             0                           74   
UK                                     56                           56   
USA                                    87                           86   

             Importance_saving_enery  Importance_owning_computer  \
Country                                                            
Australia                        165                         161   
Canada                           100                         100   
New Zealand                        0                           0   
UK                                56                          56   
USA                               63                          85   

             Importance_Internet_access  
Country                                  
Australia                           163  
Canada                              100  
New Zealand                           0  
UK                                   56  
USA                                  85

<span style="color: #006600;"><b> Answer Q1. Australia has the most interanational students. </b></span>

<b><span style="color: #FF0000;"> Q2.  Which country had the most female students ?</span></b>

In [11]:
student_data.groupby('Country')['Gender'].value_counts()

Country      Gender
Australia    Male      89
             Female    81
Canada       Female    50
             Male      50
New Zealand  Female    39
             Male      35
UK           Female    30
             Male      26
USA          Male      52
             Female    47
Name: Gender, dtype: int64

<span style="color: #006600;"><b> Answer Q2. We can see that Australia has the most female international students. </b></span>

<b><span style="color: #FF0000;"><b> Q3. What was the most used form of commuting for students ? </b></span>

In [8]:
student_data.sort_values('Country')['Travel_to_School'].value_counts()

Car                               223
Bus                               147
Walk                               95
Cycle                              16
Rail (Train/Tram/Subway)           10
Skateboard/Scooter/Rollerblade      2
Rail                                1
Other                               1
Boat                                1
Board                               1
Name: Travel_to_School, dtype: int64

<span style="color:green"><b> Answer Q3. We can see that most international students prefer to commute by car with a total of 223.</b></span>

<div align="center"><span style="color:green"><h1> THANK YOU!! </h1></span></div>